pkg/terraform/util.go

```diff
@@ -87,3 +88,40 @@ func CreateTerraformBackendFile(assetDir, data string) error {
 
 	return nil
 }
+
+// PrepareTerraformRootDir creates a directory named path including all
+// required parents.
+// An error is returned if the directory already exists.
+func PrepareTerraformRootDir(path string) error {
+	pathExists, err := util.PathExists(path)
+	if err != nil {
+		return errors.Wrapf(err, "failed to stat path %q: %v", path, err)
+	}
+
+	if pathExists {
```

Comment:
Does `os.MkdirAll` fail if called multiple times? If no, then we can simplify this function a lot.

Reply:
No, it doesnt. Hence I removed the check.